### DISCUSSION TITLE
Error on build newest source

### DISCUSSION THREAD

**Discussion by VnHoangAnh** [citizenfx/fivem#2039] [Engineering Discussion]
I encounter this error on building new source
`Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0246	The type or namespace name 'Hash' could not be found (are you missing a using directive or an assembly reference?)	CitizenFX.Core	fivem\code\client\clrcore-v2\Native\CustomNativeWrapper.cs	20	Active
`
Anyone encounter this? How to build with new mono?

**Comment by devnull9090** [comment]
I am also having this issue, did you find a solution?

**Comment by ktx-mega** [comment]
I you use boost > 1.79 you might want to downgrade. They moved the locale libs !
